_docs/management/using-ssh.md

```diff
@@ -26,6 +26,8 @@ perform debugging, environment inspection, and other tasks.
 
 This overrides the port health check with the `process` check, and sets the start command to just `sleep`. That will give you 10 minutes to `cf ssh` and inspect your container.
 
+**Error opening SSH connection**: In prior versions of the CF API (CAPI) app instances used a single process. In version 3 of the CAPI, there can be more then 1 process that makes up and app. This may occasionally cause issues when trying to SSH to your app. If a modification or update was made to an app using a v3 command or process - even if that change was made by someone else (e.g., another member of a development team) - the app's schema in the Cloud Controller might have changed to version 3, and a standard `cf ssh` may no longer work. If this occurs, you can try using `cf v3-ssh {app-name}`. By default a `cf v3-ssh` will select the `web` process.  You can also select a different process with v3-ssh using the `--process` flag
```

Comment:
```suggestion
**Error opening SSH connection**: In prior versions of the CF API (CAPI) app instances used a single process. In version 3 of the CAPI, there can be more than 1 process that makes up an app. This may occasionally cause issues when trying to SSH to your app. If a modification or update was made to an app using a v3 command or process - even if that change was made by someone else (e.g., another member of a development team) - the app's schema in the Cloud Controller might have changed to version 3, and a standard `cf ssh` may no longer work. If this occurs, you can try using `cf v3-ssh {app-name}`. By default a `cf v3-ssh` will select the `web` process.  You can also select a different process with v3-ssh using the `--process` flag
```